iOS Client/PodcastsController/PodcastsCoordinator.swift

```diff
@@ -22,6 +23,10 @@ class PodcastsCoordinator: Coordinator, StoreSubscriber {
     
     let podcastsStore = MBPodcastsStore()
     
+    var player = AVPlayer()
+    var currentPlayingPodcast: MBPodcast?
+    var playerState: PlayerState = .initialized
```

Comment:
interesting, so this is a locally cached substate

Reply:
Yeah basically whether to tell if the player is already playing when a new state comes in. (so we don't have to call player.play() again )